### PR TITLE
Feature interactive list

### PR DESCRIPTION
**Pull request by webdevmaxi**
### Add, Remove and Update functionality.
- Add items by inputting text onto the input.
- Removing items by clicking on the trash icon.
- Update item description by clicking on the text.
### Notes
- I did not create a new branch by accident, so the code is in the previous branch: Feature-Interactive-List. I'm working on moving commits to the appropriate branch.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
 import './style.css';
 import TaskList from './tasklist.js';
```
Comment (carlos-ssh) [inline src/index.js:3]: - Kindly fix issues that it's appearing in chrome console.
Screen Shot 2021-07-24 at 18 21 09

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @webdevmaxi , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (webdevmaxi) [reply src/index.js:3]: I'm sorry but that's not my code. Ironically, I forgot to add a button to clear all the completed tasks :D.

File: src/tasklist.js [hunk 2]
```diff
@@ -1,6 +1,9 @@
 import CompletedTask from './completed.js';
 import DragDropTask from './drag-drop.js';
```
Comment (carlos-ssh) [inline src/tasklist.js:2]: - Kindly fix your drag and drop functionality in your project.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @webdevmaxi, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/tasklist.js (hunk 2 above)
Comment (webdevmaxi) [reply src/tasklist.js:2]: Sure. I would need you to please specify what is wrong though. It's working on my side. I've tested it on Firefox and for some reason it doesn't work, but it's working in Chromium and I assume it works on Chrome as well.

Comment (webdevmaxi) [reply src/tasklist.js:2]: https://user-images.githubusercontent.com/86670011/126884798-a39d5c6f-4f1b-4586-a059-7c6734d80387.mp4

Here's a recording of a test done in Chromium.

Comment (Davidosky007) [reply src/tasklist.js:2]: * done 👍

Comment (webdevmaxi) [reply src/tasklist.js:2]: Does that mean I can submit the assignment?

**Comment by Davidosky007** [comment]
# Hi @webdevmaxi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
# [OPTIONAL]
## * Kindly save all changes and create tasks in the local storage, currently when i add tasks it doesn't show in the local storage and when i refresh the page everything goes away 
image

## * Am so sorry the previous reviewer forgot to point this out 🙏 as it is a requirement for this project, so i marked it optional but i would strongly suggest you fix it as it will be needed in the next project
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**